torch/nn/_functions/rnn.py

```diff
@@ -7,19 +7,22 @@
     pass
 
 
-def RNNReLUCell(input, hidden, w_ih, w_hh, b_ih=None, b_hh=None):
-    hy = F.relu(F.linear(input, w_ih, b_ih) + F.linear(hidden, w_hh, b_hh))
+def RNNReLUCell(input, hidden, w_ih, w_hh, b_ih=None, b_hh=None, skip_input=False):
+    xw_ih = input if skip_input else F.linear(input, w_ih, b_ih)
```

Comment:
haha honestly not sure what to call it, its the input (x) * hidden input weight (w_ih). Any suggestions?